frontend/src/customAxios.ts

```diff
@@ -37,10 +37,10 @@ export const CustomAxios = ({
     baseURL,
     headers: {
       'Access-Control-Allow-Origin': '*',
-      Authorization: `Bearer ${store.getState().jwt}`,
```

Comment:
This is only called during instantiation, the interceptor is called for every request.